app/lib/firebase

```diff
@@ -1,10 +1,26 @@
 /// <reference path="typings/firebase/firebase.d.ts" />
 import {Injectable} from 'angular2/core';
+import {Observable, Observer} from 'rxjs/Rx';
 
 @Injectable()
 export class FirebaseService {
-  dataRef: Firebase;
```

Comment:
Create static reference to music section of the database `static MUSIC: string = 'music';`